service/controller/resource/tccp/create.go

```diff
@@ -834,3 +856,20 @@ func (r *Resource) updateStack(ctx context.Context, cr infrastructurev1alpha2.AW
 
 	return nil
 }
+
+func mustCIDRFromSubnets(subnets []*ec2.Subnet, kind string, name string) string {
```

Comment:
what is the reason of adding kind in there?

Reply:
Kind is the subnet type. So either `private` or `public`. I use kind because `type` is a reserved keyword in go.